frontend/components/containers/comments/Comment.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <div>
+    <v-timeline-item
+      small
+    >
+      <div>
+        <div class="font-weight-normal">
```

Comment:
pretty sure there is some framework magic for setting the font weight instead of hardcoding the class.

Reply:
I remember that I took this directly from the vue doc. See: https://vuetifyjs.com/en/styles/text-and-typography/#font-emphasis.

Reply:
alright.